layouts/partials/blocks/explore-newsletters.html

```diff
@@ -0,0 +1,52 @@
+<section class="mb-[72px] w-full pt-12">
+  <div class="mx-auto max-w-prose px-4">
+    <h3 class="border-b-2 border-[#e2e2e2] pb-2 text-2xl font-bold text-g-9">
```

Comment:
There are still a couple of these left. You can find them if you search for `-[#` on the Github page.